### ISSUE TITLE
HA Bridge not detected at all

### ISSUE THREAD

**Issue by jimbaou** [bwssytems/ha-bridge#791]
Hello,

I guess my problem is very basic, but I am a newbye with HA Bridge.

I have HA Bridge running on a W10 computer. I can check it is running at IP 127.0.0.1, or browsing the W10 computer IP address from another computer.

However, it is not detected by either the Harmony software nor the HUE Android App.  

I have deactivated the firewall of the computer running HA Bridge, but it changed nothing.

I also have a true Philips HUE bridge with which I have no problem.

Thanks in advance for any help.

EDIT: here is the log I have when the HUE app looks for a bridge (the IP address is that of the Android device):

11-17-2017 13:04:36.476 | DEBUG | isSSDPDiscovery Got SSDP packet from 192.168.0.118:54901, body: M-SEARCH * HTTP/1.1  HOST:239.255.255.250:1900  ST:ssdp:all  Man:"ssdp:discover"  MX:3 | com.bwssystems.HABridge.upnp.UpnpListener
-- | -- | -- | --
11-17-2017 13:04:39.342 | DEBUG | httpMethod:get, uri: /system/logmsgs | spark.webserver.MatcherFilter
11-17-2017 13:04:39.342 | DEBUG | get params | spark.Request
11-17-2017 13:04:39.342 | DEBUG | get splat | spark.Request
11-17-2017 13:04:39.342 | DEBUG | get params | spark.Request
11-17-2017 13:04:39.342 | DEBUG | get splat | spark.Request
11-17-2017 13:04:39.342 | DEBUG | Get logmsgs.

**Comment by bwssytems** [comment]
since that portion of the log shows nothing except a discovery message was sent. Need more log info, the whole log file from the command line is the best.

**Comment by michaelhinchey** [comment]
Are you able to control the devices from the habridge?

**Comment by jimbaou** [comment]
Thanks for your interest.

I am sorry for being so basic, but where can I find the log file ? The only log I have is the one dispayed on the HA bridge web interface. This is what I have copied/pasted in my message

I have no device, my end purpose is to control Domoticz from my Harmony, and it seems this can be done with HA Bridge. I am begining with the link Harmony -> HABridge.

**Comment by jimbaou** [comment]
Very strange, indeed ....

It took me some time to find out how to log to a file. When I understood I had to start HAbridge in console mode, it took me some more time to obtain the log in DEBUG mode, as this mode is not saved in the config file. And when I was finally in a position to log to a file the connection failure in DEBUG mode, there was no more connection failure. I was able to control the HA bridge from my Harmony. I tried to step back to a failing configuration, but I couldn't. Now it works perfectly, no matter what.

So the issue is closed. But don't ask me why.

Anyway, thanks for this software, and thanks for your time.
